graylog2-web-interface/src/components/event-definitions/replay-search/EventInfoBar.tsx

```diff
@@ -0,0 +1,218 @@
+        res.push((allNotifications[cur.notification_id]));
+      }
+
+      return res;
+    }, []);
+  }, [eventDefinition, allNotifications]);
+
+  const isEDUpdatedAfterEvent = !isEventDefinition && moment(eventDefinition.updated_at).diff(eventData.timestamp) > 0;
+  const highlightingRules = useHighlightingRules();
+
+  const highlightingColors = useMemo(() => {
+    const aggregationsSet = new Set(aggregations.map(({ fnSeries, value, expr }) => `${fnSeries}${expr}${value}`));
+
+    return highlightingRules.reduce((res, rule) => {
+      const { field, value, condition } = rule;
+      const color = rule.color as StaticColor;
+      const expr = conditionToExprMapper?.[condition];
+
+      if (expr) {
+        const key = `${field}${expr}${value}`;
+
+        if (aggregationsSet.has(key)) {
+          res[key] = color.color;
+        }
+      }
+
+      return res;
+    }, {});
+  }, [aggregations, highlightingRules]);
+
+  return (
+    <FlatContentRow>
+      <Header>
+        <Button bsStyle="link" className="btn-text" bsSize="xsmall" onClick={toggleOpen}>
+          <Icon name={`caret-${open ? 'down' : 'right'}`} />&nbsp;
+          {open ? 'Less details' : 'More details'}
+        </Button>
+      </Header>
+      {open && (
+      <Container>
+        <Row>
+          {!isEventDefinition && (
+          <Item>
+            <b>Timestamp:</b>
+            <Timestamp dateTime={eventData.timestamp} />
+          </Item>
+          )}
+          {isEDUpdatedAfterEvent && (
+          <Item>
+            <b>Event definition updated at:</b>
+            <AlertTimestamp dateTime={eventDefinition.updated_at} />
+            <HoverForHelp iconSize="xs">{
+              `Event definition ${eventDefinition.title} was edited after this event happened.
+              Some of aggregations widgets might not be representative for this event.`
+            }
+            </HoverForHelp>
+          </Item>
+          )}
+          {!isEventDefinition && (
+          <Item>
+            <b>Event definition:</b>
+            <span>
+              <Link target="_blank"
+                    to={Routes.ALERTS.DEFINITIONS.show(eventDefinition.id)}>{eventDefinition.title}
+              </Link>
+            </span>
+          </Item>
+          )}
+          <Item>
+            <b>Priority: </b>
+            <span>{lodash.upperFirst(EventDefinitionPriorityEnum.properties[eventDefinition.priority].name)}</span>
+          </Item>
+          <Item>
+            <b>Execute search every:</b>
+            <span>{executeEvery.duration} {executeEvery.unit.toLowerCase()}</span>
+          </Item>
+          <Item>
+            <b>Search within:</b>
+            <span>{searchWithin.duration} {searchWithin.unit.toLowerCase()}</span>
+          </Item>
+          <Item>
+            <b>Description:</b>
+            <span>{eventDefinition.description}</span>
+          </Item>
+          <Item>
+            <b>Notifications:</b>
+            <span>
```

Comment:
Let's display an indicator, when there is no notification? 
![image](https://user-images.githubusercontent.com/46300478/221233387-09506f96-4ba8-4362-873b-9c9e4a290449.png)